tests/test_parser.py

```diff
@@ -279,5 +284,5 @@ def test_complex_type_with_all_resets_elements_max_occurs(self):
                 ],
             ),
         )
-        schema = SchemaParser.from_string(wrap(xsd))
+        schema = self.parser.from_string(wrap(xsd))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'clazz' in method call](https://app.codacy.com/manual/tefra/xsdata/pullRequest?prid=4930005)